dd-sdk-android/src/main/kotlin/com/datadog/android/rum/tracking/ActivityWhitelistAcceptAll.kt

```diff
@@ -0,0 +1,21 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.rum.tracking
+
+import android.app.Activity
+
+/**
+ * A predefined [WhitelistPredicate] which whitelists an [Activity]
+ * to be tracked as a RUM View event.
+ * This is the default behaviour for the [ActivityViewTrackingStrategy].
+ */
+class ActivityWhitelistAcceptAll : WhitelistPredicate<Activity> {
```

Comment:
If you accept all, it's not a whitelist…

Reply:
Exactly, I will correct those names.